R/deparse-data-frame.R

```diff
@@ -0,0 +1,137 @@
+#' @export
+deparse.data.frame <-
+  function(x, as_tibble = FALSE, as_tribble = FALSE, generate_mutate = TRUE, ...) {
```

Comment:
That's a good point, we can decide later. The `as_tribble` format also needs better vertical alignment.